pkg/assertutil/assertutil.go

```diff
@@ -22,20 +22,14 @@ type Checker struct {
 }
 
 // NewChecker creates Checker with FailNow function.
-func NewChecker(failNow func()) *Checker {
```

Comment:
Why remove this arg?

Reply:
Once we remove all `pingcap/check` dependencies, we can change it back.